cmd/infracost/testdata/comment_bitbucket_summary_only/comment_bitbucket_summary_only.golden

```diff
@@ -0,0 +1,11 @@
+
+## Infracost estimate: **monthly cost will increase by $40.56 (+100%) ↑**
+
+| **Project** | **Previous** | **New** | **Diff** |
+| ----------- | -----------: | ------: | -------- |
+| infracost/infracost/cmd/infraco...data/terraform_v0.14_plan.json | $40.56 | $81.12 | +$40.56 (+100%) |
+
+This comment will be updated when the cost estimate changes.
+Is this comment useful? [Yes](https://dashboard.infracost.io/feedback/redirect?runId=&value=yes), [No](https://dashboard.infracost.io/feedback/redirect?runId=&value=no), [Other](https://dashboard.infracost.io/feedback/redirect?runId=&value=other)
+
+Comment not posted to Bitbucket (--dry-run was specified)
```

Comment:
@alikhajeh1 FYI, this is an example of the comment with `--summary-only`, it will also include policy checks output if they are set up, only details are excluded.

Reply:
Maybe we should call it `--exclude-cli-output` or something if it's just that that is hidden?

Reply:
huh good idea @aliscott `--exclude-cli-output` is clearer than summary-only as @vdmgolub mentioned that policy checks are included too.